deployment/kuadrant-openshift/kustomize/prometheus/grafana-instance.yaml

```
@@ -0,0 +1,35 @@
+    spec:
+      replicas: 1
+  config:
+    auth:
+      disable_login_form: "false"
+    auth.anonymous:
+      enabled: "true"
+    log:
+      mode: console
+    security:
+      admin_password: secret
+      admin_user: root
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)